finalPrep/annotations.js

```diff
@@ -3,40 +3,66 @@
 
 //Build a Bear
 function buildABear(name, age, fur, clothes, specialPower) {
+  //variable greeting assigned to the bear
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+//variable demographics assigned to the parameters name and age
   var demographics = [name, age];
+// variable assigning a special power saying
   var powerSaying = "Did you know that I can " + specialPower + " ?";
+//variable assigned to build of the bear
   var builtBear = {
+//key value basic info
     basicInfo: demographics,
+//key value clothes
```

Comment:
It could be worth noting where values come from in this portion... do they come from a parameter? Or a variable declared above?